From a208176ca1d9eedf8aa6bf12fde6a7a9579ab549 Mon Sep 17 00:00:00 2001
From: Tim Wiederhake <twiederh@redhat.com>
Date: Mon, 22 Feb 2021 13:20:08 +0100
Subject: [PATCH] cpu_map/sync_qemu_i386.py: Add mapping for svme-addr-check

This was added to qemu in commit 5447089c2b3b084b51670af36fc86ee3979e04be.

Signed-off-by: Tim Wiederhake <twiederh@redhat.com>
Reviewed-by: Jiri Denemark <jdenemar@redhat.com>

Origin: upstream, https://libvirt.org/git/?p=libvirt.git;a=commit;h=a208176c
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1921880
Last-Update: 2021-04-16

---
 src/cpu_map/sync_qemu_i386.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cpu_map/sync_qemu_i386.py b/src/cpu_map/sync_qemu_i386.py
index 02e6724f65..3febecdfd1 100755
--- a/src/cpu_map/sync_qemu_i386.py
+++ b/src/cpu_map/sync_qemu_i386.py
@@ -142,6 +142,7 @@ def translate_feature(name):
         "CPUID_SS": "ss",
         "CPUID_SVM_NPT": "npt",
         "CPUID_SVM_NRIPSAVE": "nrip-save",
+        "CPUID_SVM_SVME_ADDR_CHK": "svme-addr-check",
         "CPUID_TSC": "tsc",
         "CPUID_VME": "vme",
         "CPUID_XSAVE_XGETBV1": "xgetbv1",
-- 
2.31.1

